extensions/AzureCosmosDBMongoDB/AzureCosmosDBMongoDB/AzureCosmosDBMongoDB.csproj

```
@@ -0,0 +1,42 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <RollForward>LatestMajor</RollForward>
+        <AssemblyName>Microsoft.KernelMemory.MemoryDb.AzureCosmosDBMongoDB</AssemblyName>
+        <RootNamespace>Microsoft.KernelMemory.MemoryDb.AzureCosmosDBMongoDB</RootNamespace>
+        <NoWarn>$(NoWarn);CA1724;CS1591;CA1308;</NoWarn>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.KernelMemory.Abstractions" Condition="'$(SolutionName)' != 'KernelMemoryDev'" />
+        <PackageReference Include="Newtonsoft.Json" />
+        <ProjectReference Include="..\..\..\service\Abstractions\Abstractions.csproj" Condition="'$(SolutionName)' == 'KernelMemoryDev'" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <PackageReference Include="System.Linq.Async" />
+        <PackageReference Include="MongoDB.Driver" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <InternalsVisibleTo Include="Microsoft.UnitTests" />
+    </ItemGroup>
+
+    <Import Project="../../../code-analysis.props" />
+
+    <Import Project="../../../nuget-package.props" />
+
+    <PropertyGroup>
+        <IsPackable>true</IsPackable>
```

```suggestion
        <IsPackable>false</IsPackable>
```
Let's keep this disabled for now, we need to do some testing after merging, and make other changes to docs and DI (no need to make them now)

---

Sure, I'll add filters logic in the next iteration of the PR, in a couple of weeks.